test/adainterop/main.map

```diff
@@ -0,0 +1,1465 @@
+Archive member included to satisfy reference by file (symbol)
```

Comment:
I guess that this file shouldn't be in the repo

Reply:
Fixed